client/src/components/ContactForm/ContactForm.jsx

```diff
@@ -0,0 +1,84 @@
+import React, { useState, useRef } from 'react';
+import * as Form from '@radix-ui/react-form';
+import './style.css';
+import emailjs from '@emailjs/browser';
+
+function ContactForm() {
+  const form = useRef();
+  const [formData, setFormData] = useState({ username: "", email: "", message: "" });
+
+  const handleChange = (event) => {
+    const { name, value } = event.target;
+    setFormData((prevFormData) => ({ ...prevFormData, [name]: value }));
+  };
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    console.log(`Username: ${formData.username}, Email: ${formData.email}, Message: ${formData.message}`);
+
+    emailjs
+      .sendForm(process.env.SERVICE_KEY, process.env.TEMPLATE_KEY, form.current, {
+        publicKey: process.env.PUBLIC_KEY,
+      })
```

Comment:
Did you update the .env file with there new variables? I don't see the file in the pull request